lib/app/item.ex

```diff
@@ -117,11 +117,12 @@ defmodule App.Item do
   @doc """
   Update an item and its associated tags
   """
-  def update_item_with_tags(%Item{} = item, attrs) do
-    item
-    |> Item.changeset_with_tags(attrs)
-    |> Repo.update()
-  end
+
+  # def update_item_with_tags(%Item{} = item, attrs) do
```

Comment:
No longer used? 💭

Reply:
Because I changed the way we add tags to the items this function needs to be redefine.
I prefer to do this change on a next PR (https://github.com/dwyl/mvp/pull/209#issuecomment-1329724616) as I will work on moving the edit feature of the item/tags to its own page

Reply:
Sounds good. 
But _please_ leave a clarifying comment with a link to the "Todo" issue next time you comment-out code. 🙏